templates/_rows_and_columns.html

```diff
@@ -1,16 +1,40 @@
-<table>
+<table id="dbresults">
```

Comment:
Does this need to be an ID? You're only supposed to have one value of ID on a page once, so I generally try to avoid it unless it's necessary, e.g. for JavaScript or anchor links.

Reply:
inherited from old code; it's fixed